src/claimtrie.cpp

```diff
@@ -1237,20 +1212,80 @@ bool CClaimTrieCache::recursiveComputeMerkleHash(CClaimTrieNode* tnCurrent, std:
     return true;
 }
 
```

Comment:
I would like to see a better name for this function recursiveComputeMerkleHashBinaryTree or some comments/docstrings on what its supposed to do.